src/Features/Core/Portable/Snippets/SnippetProviders/AbstractSnippetProvider.cs

```diff
@@ -22,8 +22,9 @@ namespace Microsoft.CodeAnalysis.Snippets
 {
     internal abstract class AbstractSnippetProvider : ISnippetProvider
     {
-        public abstract string SnippetIdentifier { get; }
-        public abstract string SnippetDescription { get; }
+        public abstract string Shortcut { get; }
+        public abstract string Title { get; }
+        public string Description => string.Format(FeaturesResources.Code_snippet_for_0, Title);
```

Comment:
Do we want a different description?

Reply:
This seems to be consistent with old snippet, should be fine